#  Project 1: Airline Delay Analysis

## Introduction

This project analyzes the arrival delays of two airlines across five destinations. 
The dataset contains delay times for **Airline A** and **Airline B** at each destination. 

## Data Loading

I began by loading the data from the CSV file using the pandas library. 

In [1]:
import pandas as pd

df = pd.read_csv('C:\\Users\\The King\Desktop\\FALL 2024\\IS 362\\Project_1\\airline_delays.csv')

print(df)

   Destination  Airline A Delay (mins)  Airline B Delay (mins)
0     New York                      12                       7
1  Los Angeles                       5                      -2
2      Chicago                      -3                      15
3      Houston                       8                       3
4      Phoenix                       4                      -1


## Data Preview

The dataset contains the following columns:
- **Destination**: The city where the flight is arriving.
- **Airline A Delay (mins)**: The delay in minutes for Airline A.
- **Airline B Delay (mins)**: The delay in minutes for Airline B.

In [2]:
df.head()

Destination  Airline A Delay (mins)  Airline B Delay (mins)
0     New York                      12                       7
1  Los Angeles                       5                      -2
2      Chicago                      -3                      15
3      Houston                       8                       3
4      Phoenix                       4                      -1

## Data Analysis

Now, I'll calculate the average delay for both airlines to compare their performances.

In [3]:
avg_delay_A = df['Airline A Delay (mins)'].mean()
avg_delay_B = df['Airline B Delay (mins)'].mean()

print(f"Average delay for Airline A: {avg_delay_A} minutes")
print(f"Average delay for Airline B: {avg_delay_B} minutes")

Average delay for Airline A: 5.2 minutes
Average delay for Airline B: 4.4 minutes


## Conclusion

Based on my analysis, **Airline A** had an average delay of `5.2` minutes, while **Airline B** had an average delay of `4.4` minutes. 

In general, I observed that:
- Airline A tends to have shorter delays compared to Airline B.
- Airline B experienced significant delays in some destinations, particularly in **Chicago**.